spec/preservation/update_moab_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe Preservation::UpdateMoab do
+  let(:pres_update_moab) { described_class.new }
+  let(:full_druid) { 'druid:bj102hs9687' }
+  let(:mock_path) { instance_double(Pathname) }
+  let(:mock_so) { instance_double(Moab::StorageObject, object_pathname: mock_path) }
+  let(:mock_new_version) { instance_double(Moab::StorageObjectVersion) }
+  let(:verification_result) { instance_double(Moab::VerificationResult) }
+
+  context '#perform' do
+    it 'calls #ingest_bag and verify_version_storage on Moab::StorageObjectVersion' do
+      allow(LyberCore::Log).to receive(:debug).with("update-moab druid:bj102hs9687 starting")
+      expect(Moab::StorageServices).to receive(:find_storage_object).with(full_druid, true).and_return(mock_so)
+      expect(mock_path).to receive(:mkpath)
+      expect(mock_so).to receive(:ingest_bag).and_return(mock_new_version)
+      allow(verification_result).to receive(:verified).and_return(true)
+      expect(mock_new_version).to receive(:verify_version_storage).and_return(verification_result)
+      pres_update_moab.perform(full_druid)
+    end
+  end
+
+  context '#update_moab' do
+    before do
+      allow(Moab::StorageServices).to receive(:find_storage_object).and_return(mock_so)
+      allow(mock_path).to receive(:mkpath)
+    end
+
+    it 'raises ItemError when new version does not pass verification' do
```

Comment:
oh dang!  I wasn't clear.  I meant for this to be a test under perform as well.  :-(

Reply:
okay! i also rebased this off master (had to do some file shuffling)